### ISSUE TITLE
Train/Test data

### ISSUE THREAD

**Issue by jianglikun** [tangwending/MLBP#1]
The data division in the article is not very clear, so I would like to consult you:

1. For the results in Table 3, the training result is the average result of all training sets in the five-fold crossover? Why is there only one test result, is it the best one?

2. For the results in Table 4, are they the test results of the Test data set in the five-fold cross-validation? Why is it described in the main text that "We randomly selected 80% from the test set to be a subset and repeated this process five times. Then, we obtained five test subsets. On these subsets, we compared MLBP with the state-of-the art multi-label methods. As shown in Table 4"? Or did you use 80% of Test to calculate 5 repetitions?
